openlibrary/templates/merge_queue/merge_queue.html

```diff
@@ -65,53 +65,70 @@ <h1>$_('Community Edit Requests')</h1>
           $ type_str = _('Author') if r['mr_type'] == 2 else _('Work')
           <tr id="mrid-$(r['id'])">
 
-            <td id="comment-cell-$(r['id'])" class="comment-cell">
+            <td class="mr-info">
               <div class="dot-wrapper"><div class="green-dot"></div></div>
-              <div>
-                <div class="comment-cell__summary">
-                  <strong>
-                    $:_('<span class="book-title">%(title)s</span>', title=work_title, type=type_str)
-                  </strong>
-                </div>
-
-                $if len(comments) > 1:
-                  <div class="comment-cell__description">
-                    <span>$_('Showing most recent comment only.')</span>
-                    <a href="javascript:;" class="comment-expand" data-target-id="hidden-comments-$r['id']">$_('View all')</a>
-                  </div>
-                <div id="hidden-comments-$r['id']" class="comment-cell__old-comments hidden">
-                  $for c in comments[:-1]:
-                    $:render_template('merge_queue/comment', comment=c)
-                </div>
-                <div class="comment-cell__newest-comment">
+              <div id="comment-cell-$(r['id'])" class="comment-cell">
+                $:_('<span class="book-title">%(title)s</span>', title=work_title, type=type_str)
+                <div id="latest-comment-$r['id']" class="comment-cell__newest-comment">
                   $if comments:
-                    $:render_template('merge_queue/comment', comment=comments[-1])
+                    $:render_template('merge_queue/newest_comment', comment=comments[-1])
                   $else:
                     <span>$_('No comments yet.')</span>
                 </div>
-                $if is_submitter or can_merge:
-                  <hr>
-                  <div class="comment-cell__input">
-                    <textarea rows="1" placeholder="$_('Add a comment...')"></textarea>
-                    <input class="mr-comment-btn" type="button" value="Reply" data-mrid="$r['id']" data-merge-type="$r['mr_type']">
+                <div id="hidden-comments-$r['id']" class="comment-cell__old-comments hidden">
+                  <div class="comment-cell__old-comments-section">
+                    $for c in reversed(comments):
+                      $:render_template('merge_queue/old_comment', comment=c)
```

Comment:
```suggestion
                    $for c in comments:
                      <div class="mr-comment">
                        <div class="mr-comment__body"><span>@$c['username']</span> $c['message']</div>
                      </div>
```
Comments shouldn't be in reverse order.

Reply:
done